config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -18,6 +18,7 @@
     <!-- Javadoc files should not have new line by design (on purpose). -->
      <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]grammars[\\/]javadoc[\\/].*\.javadoc"/>
      <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]astprinter[\\/].*\.javadoc"/>
+     <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]main[\\/].*\.javadoc"/>
```

Comment:
@romani Should we just suppress all javadoc files?
IE: `files="[\\/]test[\\/].*\.javadoc"`